sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1215,6 +1232,9 @@ nonReserved
     | YEARS
     ;
 
+//============================
+// Start of the keywords list
+//============================
 SELECT: 'SELECT';
```

Comment:
We should sort the keyword list. It's better to do it in a followup to keep the diff small for this PR.

Reply:
+1